packages/ui-icons/package.json

```diff
@@ -23,6 +23,7 @@
     "prop-types": "^15.5.10",
     "raw-loader": "^0.5.1",
     "react-svg-inline": "^2.0.0",
+    "standard": "^11.0.0",
```

Comment:
How does this work as both a peer dep and a direct dep?

Reply:
It shouldn't be a direct dep - just a devDep. I'll update it. I'm bad at typing lerna commands.